cmake/external_mlir.cmake

```diff
@@ -68,6 +68,11 @@ message(STATUS "Found LLVM ${LLVM_PACKAGE_VERSION}")
 message(STATUS "Using modules in: ${LLVM_DIR}")
 message(STATUS "LLVM RTTI is ${LLVM_ENABLE_RTTI}")
 
+set(MLIR_DIR "${MLIR_LLVM_BUILD_DIR}/lib/cmake/mlir"
+    CACHE PATH "Path to MLIR cmake modules")
```

Comment:
I would probably rename this to MLIR_CMAKE_PATH since there are quite a few var with MLIR directories already. Or if it's only used in the next line, maybe you can inline the path there?

Reply:
Done.